utils/m0singlenode

```diff
@@ -32,8 +32,8 @@ readonly base_dir=$(dirname $self)
 M0_SRC_DIR=`readlink -f $0`
 M0_SRC_DIR=${M0_SRC_DIR%/*/*}
 
-. $M0_SRC_DIR/utils/functions # m0_default_xprt
-XPRT=$(m0_default_xprt)
+source /usr/libexec/cortx-motr/motr-service.functions
```

Comment:
This script is supposed to work without installing Motr scripts/binaries into system?
We must make sure it can run from source code.

Reply:
Agree.
Let's wait until https://jts.seagate.com/browse/EOS-26866 Selection of network transport based on configure option is ready.